misc/formatter/modules/formatter-cli/src/test/java/org/ballerinalang/formatter/cli/FormatCmdTest.java

```diff
@@ -82,6 +87,30 @@ public void formatCLIOnBallerinaProject() {
         }
     }
 
+    @Test(description = "Test ballerina project formatting")
```

Comment:
it would be convenient if we can be more specific with the description, as adding `with configurations` as a suffix